privacyidea/lib/tokens/papertoken.py

```diff
@@ -107,7 +106,18 @@ def get_class_info(key=None, ret='all'):
                            "type": "int",
                            "desc": _("The number of OTP values, which are "
                                      "printed on the paper.")
+                       },
+                       ACTION.MAXTOKENUSER: {
+                           'type': 'int',
+                           'desc': _("The user may only have this maximum number of paper tokens assigned."),
+                           'group': GROUP.TOKEN
+                       },
+                       ACTION.MAXACTIVETOKENUSER: {
+                           'type': 'int',
+                           'desc': _("The user may only have this maximum number of active paper tokens assigned."),
+                           'group': GROUP.TOKEN
```

Comment:
remember my (rejected) changes from #1938? We could define template policies which will be automatically generated in each token class. We could save some code duplication. Should i implement this?

Reply:
No. I do not remember. The 1938 points to something merged.
Did we have an issue for this idea? Otherwise you can create one and I would like to have a few lines of meta code in the issue so we have a common understanding what this implementation would do.

I also do not like the many lines of code - although they are no lines of **code**.
We would still have specific ``desc`` of the policies, so I am not so sure, how some templates could help, keeping the flexibility and producing *less* lines of code.